### PR TITLE
Added Contributions flow  behind experiments flag

### PR DESCRIPTION
**Pull request by dparikh**

### CODE REVIEW

File: src/runtime/runtime-test.js [hunk 1]
```diff
@@ -1168,6 +1171,20 @@ describes.realWin('ConfiguredRuntime', {}, env => {
     });
   });
 
+  it('should call "showContrinutionOptions" with options', () => {
```
Comment (chenshay) [inline src/runtime/runtime-test.js]: showContrinutionOptions -> showContributionOptions

File: src/api/subscriptions.js [hunk 2]
```diff
@@ -96,6 +96,12 @@ export class Subscriptions {
    */
   showAbbrvOffer(opt_options) {}
 
+  /**
+   * Show contribution options.
+   * @param {!OffersRequest=} opt_options
+   */
+  showContributionOptions(opt_options) {}
```
Comment (chenshay) [inline src/api/subscriptions.js]: the term "options" is really confusing. Please make it more clear what is going in here

File: src/runtime/runtime-test.js (hunk 1 above)
Comment (dparikh) [reply src/runtime/runtime-test.js]: Done

File: src/api/subscriptions.js (hunk 2 above)
Comment (dparikh) [reply src/api/subscriptions.js]: This shows the contribution options to the users to select from. 
Added more descriptive comments.

File: src/api/subscriptions.js [hunk 3]
```diff
@@ -96,6 +96,16 @@ export class Subscriptions {
    */
   showAbbrvOffer(opt_options) {}
 
+  /**
+   * Show contribution options for the users to select from.
+   * The options are grouped togather by periods (Weekly, Monthly, etc.).
```
Comment (chenshay) [inline src/api/subscriptions.js]: togather -> together

Comment (dparikh) [reply src/api/subscriptions.js]: Done

**Pull request merged**